usbsdmux/usb2642i2c.py

```diff
@@ -361,24 +359,15 @@ def write_config(self, data):
                                        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,\
                                        0x00, 0x00, 0x00], 16)
 
-    # First looked like the data block was prefixed with some magic.
-    # But at a second look this was not the case. Setting the length of this
-    # field to 0 bytes prevents any padding.
-    data_prefix = list_to_uint8_array([], 0)
-
```

Comment:
I would prefer keeping this comment in some form, as it is the result of the tedious reverse-engineering of the vendor-tool.